modules/systemtests/src-test/org/ejbca/ui/cli/ca/CaImportCertDirCommandTest.java

```diff
@@ -1,19 +1,9 @@
-/*************************************************************************
- *                                                                       *
  *  EJBCA Community: The OpenSource Certificate Authority                *
```

Comment:
add back header

Reply:
Sorry about this. Not sure how this happened. Will fix.